py/desisim/scripts/quickquasars.py

```diff
@@ -73,14 +74,20 @@ def parse(options=None):
 
     return args
 
-def simulate_one_healpix(ifilename,args,model,obsconditions,decam_and_wise_filters,footprint_healpix_weight,footprint_healpix_nside,seed,bal=None) :
+
+def get_pixel_seed(pixel, nside, global_seed):
+    if global_seed is None:
+        # return a random seed
+        return np.random.randint(2**32, size=1)[0]
+    np.random.seed(global_seed)
+    seeds = np.random.randint(2**32, size=healpy.nside2npix(nside))
```

Comment:
You can have repeated seeds.
Better do:
`seeds = sp.unique(np.random.randint(2**32, size=10*healpy.nside2npix(nside)))[:healpy.nside2npix(nside)]`

Reply:
Why would you have repeat seeds? I'm confused.

Reply:
`np.random.randint` can give you twice the same seed. The probability is low because `2**32` is a big number, but see the examples at the bottom: https://docs.scipy.org/doc/numpy/reference/generated/numpy.random.randint.html

Reply:
I'm on my way home now, will implement these later